config/config.go

```diff
@@ -34,20 +35,21 @@ func Get() (*Config, error) {
 	}
 
 	cfg = &Config{
-		BindAddr:                   ":29200",
-		GracefulShutdownTimeout:    5 * time.Second,
-		HealthCheckInterval:        30 * time.Second,
-		HealthCheckCriticalTimeout: 90 * time.Second,
-		OTBatchTimeout:             5 * time.Second,
-		OTExporterOTLPEndpoint:     "localhost:4317",
-		OTServiceName:              "dp-legacy-cache-proxy",
-		BabbageURL:                 "http://localhost:8080",
-		LegacyCacheAPIURL:          "http://localhost:29100",
-		CacheTimeDefault:           15 * time.Minute,
-		CacheTimeErrored:           30 * time.Second,
-		CacheTimeLong:              4 * time.Hour,
-		CacheTimeShort:             10 * time.Second,
-		PublishExpiryOffset:        3 * time.Minute,
+		BindAddr:                     ":29200",
+		GracefulShutdownTimeout:      5 * time.Second,
+		HealthCheckInterval:          30 * time.Second,
+		HealthCheckCriticalTimeout:   90 * time.Second,
+		OTBatchTimeout:               5 * time.Second,
+		OTExporterOTLPEndpoint:       "localhost:4317",
+		OTServiceName:                "dp-legacy-cache-proxy",
+		BabbageURL:                   "http://localhost:8080",
+		LegacyCacheAPIURL:            "http://localhost:29100",
+		CacheTimeDefault:             15 * time.Minute,
+		CacheTimeErrored:             30 * time.Second,
+		CacheTimeLong:                4 * time.Hour,
+		CacheTimeShort:               10 * time.Second,
+		PublishExpiryOffset:          3 * time.Minute,
+		IsPublishExpiryOffsetEnabled: false,
```

Comment:
Really small one, but since you've put this one between `CacheTimeShort` and `PublishExpiryOffset` in other files, maybe you could do the same here.